data/json/items/gun/545x39.json

```diff
@@ -36,24 +36,23 @@
     "price": 420000,
     "bashing": 11,
     "to_hit": -1,
-    "material": ["steel", "plastic"],
+    "material": [ "steel", "plastic" ],
     "symbol": "(",
     "color": "brown",
     "skill": "rifle",
     "ammo": "545x39",
     "dispersion": 35,
     "sight_dispersion": 50,
-    "aim_speed" : 4,
+    "aim_speed": 4,
     "recoil": -10,
     "durability": 7,
-    "barrel_length" : 1,
```

Comment:
Correction, It has a muzzle break, and it doesn't have much to saw off due to the gas mechanism